.github/workflows/docker_build.yml

```diff
@@ -92,10 +76,7 @@ jobs:
           ATLANTIS: ${{ matrix.ATLANTIS }}
           TF_TG: ${{ matrix.TF_TG }}
 
-      # ------------------------------------------------------------
-      # Build, Test and Push
-      # ------------------------------------------------------------
-      - name: Build
+      - name: Build and push
```

Comment:
This task is not pushing, why was it renamed?

Reply:
Agree. I will fix it.